samples/README.md

```diff
@@ -253,6 +254,25 @@ __Usage:__
 
 
 
+### Create a Cloud Storage subscription
+
+Demonstrates how to create a subscription with Cloud Storage.
+
+View the [source code](https://github.com/googleapis/nodejs-pubsub/blob/main/samples/createSubscriptionWithCloudStorage.js).
+
+[![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-pubsub&page=editor&open_in_editor=samples/createSubscriptionWithCloudStorage.js,samples/README.md)
+
+__Usage:__
+
+
+`node createSubscriptionWithCloudStorage.js <topic-name> <subscription-name> <bucket> <filenamePrefix> <filenameSuffix> <maxDuration>`
```

Comment:
Nit: why do we write "topic-name" and "subscription-name" (kebab-case) but "filenamePrefix" and "maxDuration" (camelCase)? Perhaps we should make these consistent?

Reply:
Good catch!